python/cusignal/filtering/filtering.py

```diff
@@ -165,10 +166,11 @@ def lfiltic(b, a, y, x=None):
     return zi
 
 
-def lfilter(b, a, x):
+def lfilter(
+    b, a, x, clamp=False, cp_stream=cp.cuda.stream.Stream(null=True),
```

Comment:
Very minor, but let's change 'clamp' to 'clip'; this follows the signal processing nomenclature better.

Reply:
fixed.